exporter/elasticsearchexporter/config.go

```diff
@@ -131,8 +131,12 @@ type DiscoverySettings struct {
 // all events already serialized into the send-buffer.
 type FlushSettings struct {
 	// Bytes sets the send buffer flushing limit.
+	// WARNING: This configuration is ignored. Use `flush.min_documents` instead.
 	Bytes int `mapstructure:"bytes"`
 
+	// MinDocuments configures the minimum number of documents in the send buffer to trigger a flush.
+	MinDocuments int `mapstructure:"min_documents"`
```

Comment:
Should we also add `MaxDocuments` now that we are using the pdata models?

Reply:
done